src/pages/@owner/estimate/@section/basic-info-section/Etc.tsx

```
@@ -0,0 +1,25 @@
+import FormLayout from '@components/layout/form-layout/FormLayout';
+import ErrorText from '@form/error-text/ErrorText';
+import Textarea from '@components/ui/text-area/Textarea';
+import { ESTIMATE_MAX_LENGTH } from '@pages/@owner/estimate/constants/estimate';
+
+interface EtcProps {
+  etc: string;
+  updateEtc: (etc: string) => void;
```

ESLint

[error] 'etc' is defined but never used. Allowed unused args must match /^_/u.

(no-unused-vars)